Plugins/Json/include/Acts/Plugins/Json/ITrackingGeometryJsonDecorator.hpp

```diff
@@ -0,0 +1,40 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2017-2021 CERN for the benefit of the Acts project
```

Comment:
Very pedantic comment, but in the new files, let's keep the copyright up to date

Reply:
fixed. Thanks !